portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -12,26 +12,23 @@
 @WebServlet("/comments")
 public class CommentServlet extends HttpServlet {
 
-  private Conversation converse = new Conversation();
+  private Conversation conversation = new Conversation();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("application/json");
-    String json = new Gson().toJson(converse);
+    String json = new Gson().toJson(conversation);
     response.getWriter().println(json);
   }
 
-  @Override
+    @Override
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // If the user sends another POST request after the game is over, then start a new game.
-    String username = getUsername(request);
-    String comment = getComment(request);
-
-    Comment comm = new Comment(username, comment, false);
-
-    converse.addComment(comm);
+        String username = getUsername(request);
+        String commentText = getComment(request);
+        Comment comment = new Comment(username, commentText);
```

Comment:
Suggest just inline to

Comment comment = new Comment(getUsername(request), getComment(request));

Reply:
Added inline